test/support/test_offset_utils.h

```diff
@@ -117,9 +140,20 @@ test_unwrap(UniverseContainer u)
         std::cout << "ERROR: Unwrapped selection type is equal to the actual selection type\n";
     }
 
+    //Checking if the unwrapped selection type is the same as policy::resource_type
+    typename oneapi::dpl::experimental::policy_traits<Policy>::resource_type resource_test{};
+    if(!std::is_same_v<decltype(selection_unwrap),decltype(resource_test)>)
```

Comment:
Is it necessary to use decltype(resource_test) instead of just oneapi::dpl::experimental::policy_traits<Policy>::resource_type? Seems like the construction of resource_test is not necessary.

Reply:
Fixed